<launch>

<node pkg = "rectrial" name ="basler_camera_node"   	type= "basler_camera_node" />
<node pkg = "rectrial" name ="experiment_controler_node"   type= "experiment_controller"/>

</launch>
